helm/values.yaml

```diff
@@ -24,7 +24,7 @@ wagtail:
     repository: "ghcr.io/ietf-tools/www"
     pullPolicy: IfNotPresent
     # Overrides the image tag whose default is the chart appVersion.
-    tag: "v2.0.0"
+    tag: "v2.1.3"
```

Comment:
Don't set the `tag` let this default to `appVersion`.
At the time of publishing `appVersion` gets the latest tag.
See https://github.com/ietf-tools/www/blob/main/.github/workflows/build.yml#L101

Reply:
Thanks! Commented out `tag`.